analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Integrate molecular subtyping results"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+author: Krutika Gaonkar for D3b
+date: 2020
+params:
+   is_ci: FALSE
+---
+
+The purpose of this notebook is to integrate molecular subtyping results from 
+[molecular-subtyping-pathology](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-pathology) with `pbta-histologies-base.tsv`.
+
+Here we are using v18 `pbta-histologies-base.tsv`
+
+
+## Set up
+
+```{r}
+library(tidyverse)
+data_dir <- "../../data/release-v18-20201123/"
+base_histology <- read_tsv(file.path(data_dir,"pbta-histologies-base.tsv"),
+                           col_types = readr::cols(molecular_subtype = readr::col_character(),
+                                                   short_histology = readr::col_character(),
+                           broad_histology = readr::col_character(),
+                           Notes = readr::col_character())) %>%
+  unique()
+```
+
+### Read molecular-subtyping-pathology results
+
+Reading molecular_subtype,	integrated_diagnosis, short_histology,	broad_histology and	Notes from `compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv` 
```

Comment:
This spacing is odd. Is there a reason behind this odd spacing?

Reply:
If you want to do a quick thing to try to get rid of some of the weird spacing in this notebook, you can run `styler::style_file("analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd")` and see if styler can fix some of these things for you. Beyond that, I don't know that we need to fuss with it. 
...
I just checked, and unfortunately we do not have the `styler` package in the docker container. So don't worry about this, but it could be handy for you elsewhere.

Reply:
hmm not sure where these spaces came from , checking

Reply:
the spaces looked fine on localhost, I edited it and looks ok here as well. Let me know if I missed any weird spaces